src/Writer/CSV/Writer.php

```diff
@@ -50,7 +50,13 @@ protected function openWriter(): void
     protected function addRowToWriter(Row $row): void
     {
         $cells = array_map(static function (Cell\BooleanCell|Cell\EmptyCell|Cell\NumericCell|Cell\StringCell|Cell\FormulaCell $value): string {
-            return (string) $value->getValue();
+            $value = $value->getValue();
+
+            if (false === $value) {
+                return '0';
+            }
+
+            return (string) $value;
```

Comment:
I'd rather check for types instead of values:
```suggestion
            if ($value instanceof Cell\BooleanCell) {
                return (string) (int) $value->getValue();
            }

            return (string) $value->getValue();
```